code/src/Survey.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import Summary from './Summary';
+
+// all states
```

Comment:
Very good that you comment on your code! 😄  Makes it easier for me and you to understand 😉